### ISSUE TITLE
Error when clicking to terms pages

### ISSUE THREAD

**Issue by nwdls** [meigwilym/cytarse#89]
The /terms pages aren't showing when clicked from the homepage, so can't check formatting of T&Cs

**Comment by nwdls** [comment]
Still getting an error. Same error for /holi and /cysylltu too.

**Comment by MattHall** [comment]
I've just taken a look at the terms page, and it's a bit of a mess - lots of blocks like this:
```
= succeed "." do
    b Producers
  | For the
```

How has this page been generated?

**Comment by nwdls** [comment]
@MattHall Um...I use a html to slim converter to get a bunch of words on there as quickly and roughly as possible. Seems like it was too rough... 

I did the same for the Privacy Policy. I was gonna ask somebody to clean up the code after I'd had a go. Maybe best to leave that to somebody else.

**Comment by MattHall** [comment]
Do you have the pages as HTML? If so, we can go with that if it's easier?

**Comment by nwdls** [comment]
No, they were in Word originally then I found a way to get the in rough HTML...

I'll start again. ;)
